src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -71,7 +80,10 @@ Profile::Profile() :
     _scrollbarState{ },
     _closeOnExit{ true },
     _padding{ DEFAULT_PADDING },
-    _icon{ }
+    _icon{ },
+    _backgroundImage{ },
+    _backgroundImageOpacity{ },
+    _backgroundImageStretchMode{ }

```

Comment:
@d-bingham It is unnecessary to initialize the added _backgroundXXXX methods here.   They are already initialized to the "empty state" when they are declared in the header.

Reply:
Strictly speaking it's not necessary, and style guidelines differ on whether to include them or not.  They're in there following the precedent in the rest of the profile code which explicitly initializes members with default constructors.

Reply:
@d-bingham you are correct that the style guides differ!  @binarycrusader what is the Windows Terminal's team thoughts on this?  I ping'd you because I _think_ you may have mentioned this in reviews.   Knowing what the expectations are would be helpful when reviewing.  (ie - if this is cool then we can stop looking for it)

Reply:
I’m not on the Terminal team, merely an interested party that contributes.  However, I suspect the general idea is that new code generally conforms to the C++ guidelines and modifications to existing try to match existing style.

Reply:
@binarycrusader's quite correct. There's a balance we're still trying to strike here between what is _correct_ and what is _complete_ :smile: Thanks for bearing with us!

Reply:
@DHowett-MSFT No worries.   I am trying to find the correct balance when reviewing the code.  I don't want to mention stuff like this if it is not appropriate.    It appears the recommendations are being worked on, so all is good.    In the meantime - what is your opinion on commenting on stuff like this?  Should I hold off or continue to point these out?

Reply:
I think it's helpful for you to point these things out, but it's even more helpful if we start to accumulate `Area-CodeHealth` issues for things like it :smile:

Reply:
@DHowett-MSFT Sounds like a plan.